.github/workflows/reusable_receptorctl_tests.yml

```diff
@@ -0,0 +1,54 @@
+---
+name: receptorctl
+
+"on":
+  workflow_call:
+
+jobs:
+  receptorctl:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version:
+        # NOTE: The highest and the lowest versions come
+        # NOTE: first as their statuses are most likely to
+        # NOTE: signal problems early:
+        - 3.11
+        - 3.8
+        - >-
+          3.10
+        - 3.9
+
+    env:
+      FORCE_COLOR: 1
```

Comment:
This one should colorize the output, which makes the CI log nicer. But I recommend playing with the rest of vars from https://github.com/ansible/pylibssh/blob/57e2802/.github/workflows/ci-cd.yml#L68-L73 / https://github.com/cherrypy/cheroot/blob/3591a1c/.github/workflows/ci-cd.yml#L61-L67 to see if there's more improvements from other vars. I know that there's a separate one for pre-commit, for example.

Reply:
I like the idea of standardizing our approach